doc/team.rst

```diff
@@ -0,0 +1,104 @@
+.. title:: Team
+
+PyGMT Team Gallery
+==================
+
+We are an international team dedicated to building a Pythonic API for the
+Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
+new and experienced users by creating user-friendly interfaces with the GMT
+C API, supporting rich display in Jupyter notebooks, and integrating with
+the PyData ecosystem.
+
+All our welcome to become involved with the PyGMT project! For more information
+about how to get involved, see the :doc:`contributing`.
+
+Distinguished Contributors
+--------------------------
+
+PyGMT Distinguished Contributors are recognized for their substantial
+contributions to PyGMT, which may include code, documentation, pull request
+review, triaging, forum responses, community building and engagement,
+outreach, and inclusion and diversity. New Distinguished Contributors are
+selected bi-annually by those listed below.
```

Comment:
> selected bi-annually by those listed below.

I might miss some discussions, but have you reached an agreement about the frequency? Do you mean "once every two years" or "twice per year"? As a non-native English speaker, it sounds like "once every two years", which is too long for an active project like PyGMT.

When I search for "bi-annually", I found this (https://www.merriam-webster.com/dictionary/biannual):

> When we describe something as biannual, we can mean either that it occurs twice a year or that it occurs once every two years. So how does someone know which particular meaning we have in mind? Well, unless we provide them with a contextual clue, they don't.

Reply:
Thanks for pointing out the confusing language. I meant twice per year and updated to specify that. It's open for discussion, I just selected 2x per year as an arbitrary starting point.